commit c130a9a3730a5ee2ab7373cbdfb2d8dd41e2e53c
Author: John Greb <h3x4m3r0n@gmail.com>
Date:   Sun May 6 20:01:57 2018 +0000

    usb: gadget ethernet: Re-enable Jumbo frames.
    
    Commit <eea52743eb5654ec6f52b0e8b4aefec952543697> upstream
    
    Fixes: <b3e3893e1253> ("net: use core MTU range checking")
    which patched only one of two functions used to setup the
    USB Gadget Ethernet driver, causing a serious performance
    regression in the ability to increase mtu size above 1500.
    
    Signed-off-by: John Greb <h3x4m3r0n@gmail.com>
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>

diff --git a/drivers/usb/gadget/function/u_ether.c b/drivers/usb/gadget/function/u_ether.c
index 6fcda62f55ea..1000d864929c 100644
--- a/drivers/usb/gadget/function/u_ether.c
+++ b/drivers/usb/gadget/function/u_ether.c
@@ -844,6 +844,10 @@ struct net_device *gether_setup_name_default(const char *netname)
 	net->ethtool_ops = &ops;
 	SET_NETDEV_DEVTYPE(net, &gadget_type);
 
+	/* MTU range: 14 - 15412 */
+	net->min_mtu = ETH_HLEN;
+	net->max_mtu = GETHER_MAX_ETH_FRAME_LEN;
+
 	return net;
 }
 EXPORT_SYMBOL_GPL(gether_setup_name_default);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1194_c130a9a3730a5ee2ab7373cbdfb2d8dd41e2e53c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1194_c130a9a3730a5ee2ab7373cbdfb2d8dd41e2e53c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
